docs/EnterpriseScaleAnalytics-CreateRepository.md

```diff
@@ -1,6 +1,6 @@
 # Data Product Analytics - Create repository from the template
 
-First, you must generate your own respository based off this template respository. To do so, please follow the steps below:
+First, you must generate your own repository based off this template repository. To do so, please follow the steps below:
```

Comment:
Change "generate" to "create"

Reply:
This must be then changed in all repos and not just this one. This should be consistent.